backend/src/main/java/com/htecgroup/skynest/security/SecurityConstants.java

```diff
@@ -0,0 +1,15 @@
+package com.htecgroup.skynest.security;
+
+public class SecurityConstants {
+
+    private SecurityConstants(){
```

Comment:
Since you already use Lombok, check how a private constructor can be done with it.